### PR TITLE
[Improvement](rpc) set grpc channel's keepAliveTime and remove proxy on InterruptedExcep…

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
## Proposed changes
1. set grpc channel's keepAliveTime
2. remove proxy on InterruptedException/TimeoutException to avoid channel unavailable

### CODE REVIEW

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

File: fe/fe-common/src/main/java/org/apache/doris/common/Config.java [hunk 1]
```diff
@@ -1465,6 +1465,12 @@ public class Config extends ConfigBase {
     @ConfField
     public static int grpc_threadmgr_threads_nums = 4096;
 
+    /**
+     * sets the time without read activity before sending a keepalive ping
+     */
+    @ConfField
```
Comment (morningman) [inline fe/fe-common/src/main/java/org/apache/doris/common/Config.java:1471]: add `description` field and give some guidance

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by w41ter — COMMENTED** [review]
Reset LGTM

**Comment by BiteTheDDDDt** [comment]
run buildall

**Comment by BiteTheDDDDt** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by w41ter — APPROVED** [review]

**Pull request merged**